resources/assets/scripts/app/directives/colorbox.coffee

```diff
@@ -0,0 +1,24 @@
+# Pony.fm - A community for pony fan music.
+# Copyright (C) 2015 Peter Deltchev
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+`angular.module('ponyfm').directive('colorbox', function() {
+  return {   
+    restrict: 'AC',    
+    link: function (scope, element, attrs) {        
+      $(element).colorbox(attrs.colorbox);     
+    }
+  };  
+});` 
```

Comment:
There should be no backtick here, and there should be a newline at the end of the file (the `.editorconfig` file in the repo root normally ensures this).

Also - this looks like JavaScript even though it's in a CoffeeScript file. Was it copy-pasted from somewhere?